config/webpack.css.config.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-env node */
+/* eslint import/no-nodejs-modules:0 */
+
+import config from '../webpack.config';
+
+// Select all CSS rules from the base webpack config
+const cssRules =
+  config?.module?.rules?.filter(
+    ruleSet =>
+      typeof ruleSet === 'object' &&
+      ['/\\.css/', '/\\.less$/'].includes(ruleSet.test?.toString() ?? '')
+  ) ?? [];
+
+config.entry = {
+  sentry: 'less/jest-ci.less',
+};
+
+config.module = {
+  ...config?.module,
```

Comment:
```suggestion
  ...(config?.module ?? {}),
```

I usually get an error if I try to `...undefined`

Reply:
```
Welcome to Node.js v12.19.0.
Type ".help" for more information.
> {...undefined}
{}
```

Actually it's totally fine believe it or not. It doesn't work for array spread however.